FoodOrderingApp-service/src/main/java/com/upgrad/FoodOrderingApp/service/entity/AddressEntity.java

```diff
@@ -0,0 +1,118 @@
+package com.upgrad.FoodOrderingApp.service.entity;
+
+//import org.apache.commons.lang3.builder.ToStringExclude;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+
+/**
+ * AddressEntity class contains all the attributes to be mapped to all the fields in 'address' table in the database
+ */
+
+public class AddressEntity implements Serializable{
```

Comment:
`@Entity` and `@Table(name = "address")` annotations are not added on `AddressEntity`, please add them.

Reply:
Added